spec/menu_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'menu'
+
+describe Menu do
+  let(:order) {{:steak=>1,:chips=>3,:pizza=>2}}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)